sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -1421,6 +1421,20 @@ def _add_argparse_args(cls, parser):
         ' directly, rather than starting up a job server.'
         ' Only applies when flink_master is set to a'
         ' cluster address.  Requires Python 3.6+.')
+    parser.add_argument(
```

Comment:
These mirror the existing [FlinkRunnerOptions.java](https://github.com/apache/beam/blob/master/runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkPipelineOptions.java#L66).